core/templates/services/audio-player.service.ts

```diff
@@ -95,6 +95,15 @@ angular.module('oppia').factory('AudioPlayerService', [
       }
     };
 
+    var _forward = function(seconds) {
+      if (_currentTrack) {
+        var currentSeconds = _currentTrack.progress * _currentTrack.duration;
+        var forwardedProgress =
+            (currentSeconds + seconds) / _currentTrack.duration;
```

Comment:
Have you considered the case where (currentSeconds + seconds) > _currentTrack.duration ?

Reply:
Yes, the audio player simply stops.

Reply:
But perhaps better to be accurate in the code (i.e. set forwardedProgress to 1.0 in that case)? You can use Math.max or similar on the numerator of the fraction in order to do this cleanly.

Reply:
Hey @seanlip By setting forwardedProgress to 1 do you mean that the audio should move to the beginning?